### ISSUE TITLE
Simple tone prompt for wrong direction

### ISSUE THREAD

**Issue by wowatatarin** [organicmaps/organicmaps#9008]
While cycling I use headphones to follow the voice commands, and phone sits in a pocket. But when i miss an intersection, it can go indefinitely, without new prompt.
For example:
![Screenshot_2024-08-18-14-28-43-313_app organicmaps](https://github.com/user-attachments/assets/8b8f660f-aac6-4275-98dd-a36aa03a7460)
I just missed a turn right and there was no signal for me to stop, check map, just the same if i would follow the right route.

The solution i would appreciate it to add 2-3 simple, tone prompts with different meaning: 
- route has changed to adapt to your current location, happens when you chose wrong way (just like Google maps, and other apps do), and app just creates a new route for you
- youre riding in wrong direction than expected by route, you should have  turn around (just like me on the screen shot) 
- you followed the last intersection just according to plan (optional, only for headphone users and partially solved by previous two)

These are not intended to be voice commands, but simple tone / prompt, maybe vibration. Just to make user check what is going on, and to not have check phone so often, just to assure, that you're following the right path.

I miss it from Google maps, where is a simple prompt, or from garmin navigation which always was talking "recalculation" , but that last one was iritating, but it was doing the job at least.

**Comment by biodranik** [comment]
@zyphlar is there a route recalculation event in such cases? Should #5303 help here?

**Comment by zyphlar** [comment]
@biodranik yep! Not really an event but the state can be calculated. I'm not sure how bicycle riding the wrong way would work, since there's no turns involved the "recalculating" may appear differently. Would need testing.

**Comment by zyphlar** [comment]
Fixed by #6576
